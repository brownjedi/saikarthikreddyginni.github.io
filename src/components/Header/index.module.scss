.header {
  background: blue;
  margin-bottom: 1.45rem;
}
