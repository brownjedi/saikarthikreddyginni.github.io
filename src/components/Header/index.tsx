import * as React from 'react'
import Link from 'gatsby-link'
import * as styles from './index.module.scss'

const Header: React.SFC<any> = () => (
  <div
    className={styles.header}
  >
    <div
      style={{
        margin: '0 auto',
        maxWidth: 960,
        padding: '1.45rem 1.0875rem',
      }}
    >
      <h1 style={{ margin: 0 }}>
        <Link
          to="/"
          style={{
            color: 'white',
            textDecoration: 'none',
          }}
        >
          Gatsby
        </Link>
      </h1>
    </div>
  </div>
)

export default Header
