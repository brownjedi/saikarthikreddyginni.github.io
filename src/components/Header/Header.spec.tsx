import * as React from 'react'
import { shallow } from 'enzyme'

import Header from '.'

describe('Header component', () => {
  it('renders correctly', () => {
    const rendered = shallow(
      <Header />
    )

    expect(rendered).toMatchSnapshot()
  })

  it('tests a truthy function', () => {
    expect('Hello').toBeTruthy()
  })
})
