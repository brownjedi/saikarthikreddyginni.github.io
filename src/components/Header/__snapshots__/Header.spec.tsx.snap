// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        style={
          Object {
            "margin": "0 auto",
            "maxWidth": 960,
            "padding": "1.45rem 1.0875rem",
          }
        }
      >
        <h1
          style={
            Object {
              "margin": 0,
            }
          }
        >
          <GatsbyLink
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
            to="/"
          >
            Gatsby
          </GatsbyLink>
        </h1>
      </div>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          style={
            Object {
              "margin": 0,
            }
          }
        >
          <GatsbyLink
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
            to="/"
          >
            Gatsby
          </GatsbyLink>
        </h1>,
        "style": Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <GatsbyLink
            style={
              Object {
                "color": "white",
                "textDecoration": "none",
              }
            }
            to="/"
          >
            Gatsby
          </GatsbyLink>,
          "style": Object {
            "margin": 0,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Gatsby",
            "style": Object {
              "color": "white",
              "textDecoration": "none",
            },
            "to": "/",
          },
          "ref": null,
          "rendered": "Gatsby",
          "type": [Function],
        },
        "type": "h1",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          style={
            Object {
              "margin": "0 auto",
              "maxWidth": 960,
              "padding": "1.45rem 1.0875rem",
            }
          }
        >
          <h1
            style={
              Object {
                "margin": 0,
              }
            }
          >
            <GatsbyLink
              style={
                Object {
                  "color": "white",
                  "textDecoration": "none",
                }
              }
              to="/"
            >
              Gatsby
            </GatsbyLink>
          </h1>
        </div>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            style={
              Object {
                "margin": 0,
              }
            }
          >
            <GatsbyLink
              style={
                Object {
                  "color": "white",
                  "textDecoration": "none",
                }
              }
              to="/"
            >
              Gatsby
            </GatsbyLink>
          </h1>,
          "style": Object {
            "margin": "0 auto",
            "maxWidth": 960,
            "padding": "1.45rem 1.0875rem",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <GatsbyLink
              style={
                Object {
                  "color": "white",
                  "textDecoration": "none",
                }
              }
              to="/"
            >
              Gatsby
            </GatsbyLink>,
            "style": Object {
              "margin": 0,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Gatsby",
              "style": Object {
                "color": "white",
                "textDecoration": "none",
              },
              "to": "/",
            },
            "ref": null,
            "rendered": "Gatsby",
            "type": [Function],
          },
          "type": "h1",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
